CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 1.26.0
+
+This minor release contains bugfixes and other improvements. It also adds better support for the arm architecture to rkt, so that you can now fetch images via autodiscovery and have the correct seccomp whitelist to run them. Also notable is the new possibilty to pass extra kernel parameters to kvm, and last but not least a significant prepare/run speedup in stage0.
+
+## New Features
+- stage1: improve duplicate mount-volume detection ([#3666](https://github.com/rkt/rkt/pull/3666)). Breaking change: volumes with duplicate names are now rejected.
```

Comment:
I would highlight this breaking change in the above summary as well, by adding a last sentence there `This also introduces stricter validation on volume names, now rejecting duplicate ones.`